<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MayDay2048</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gg.qrc">
    <normaloff>:/wegame/sx.ico</normaloff>:/wegame/sx.ico</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>上</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>左</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>右</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>下</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>分数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>300</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="show1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show15">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show14">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show12">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show16">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show11">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="show10">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gg.qrc"/>
 </resources>
 <connections/>
</ui>
